source/windows/thread.c

```diff
@@ -8,6 +8,7 @@
 #include <aws/common/linked_list.h>
 #include <aws/common/logging.h>
 #include <aws/common/private/thread_shared.h>
+#include <aws/common/string.h>
 
 #include <Windows.h>
 
```

Comment:
If aws_thread_options s_default_options is used from below and its .name isn't set, it might contain garbage as the thread name. Should we zero it out since it's not a pointer?